service/service.go

```diff
@@ -22,9 +23,20 @@ type Service struct {
 	shutdownTimeout time.Duration
 }
 
+// MessageWriter writes publishedContent events as messages
```

Comment:
I don't think lines 26 to 33 are needed anymore? No need to write things to temporary file on disk